### DISCUSSION TITLE
Compatibility with Markdown-Field-Plugin

### DISCUSSION THREAD

**Discussion by metakolor** [kirby-tools/kirby-content-translator#4] [Ideas]
Hej, 
I really love your plugin. But are there any plans to make this work with 
https://github.com/fabianmichael/kirby-markdown-field ?

Thanks in Advance

**Comment by johannschopplich** [comment]
Hi there, sorry for the late reply! I missed your request. Yes, the plugin should handle markdown fields by default. I agree.

Please upgrade to [Kirby Content Translator v2.5.0](https://kirby.tools/docs/content-translator/changelog#whats-new) for added `type: markdown` field support.
